defaults/main.yml

```diff
@@ -43,6 +43,13 @@ siteName: "io"
 nodeBase: "{{ siteName }}"
 nodeGpuBase: "gpu"
 
+# Have systemd restart slurm daemons if they fail to start
+slurm_systemd_override_slurmd: True
+slurm_systemd_override_slurmdbd: False
+slurm_systemd_override_slurmctld: False
```

Comment:
Yeah we can do both of these.
My fear was that if one starts the daemon and it's doing something badly then that would happen several times perhaps making something worse. But maybe this is unlikely?